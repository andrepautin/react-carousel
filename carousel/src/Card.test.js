import React from "react";
import { render } from "@testing-library/react";
import Card from "./Card";
import image1 from "./image1.jpg"

it("renders card without crashing", function() {
render(<Card src={ image1 } caption= "Photo by Richard Pasquarella on Unsplash" />);
});

it("matches snapshot of card", function() {
  const {container} = render(<Card src={ image1 } caption= "Photo by Richard Pasquarella on Unsplash" />);
 
  expect(container).toMatchSnapshot();
});