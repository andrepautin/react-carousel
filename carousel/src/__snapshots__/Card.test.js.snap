// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot of card 1`] = `
<div>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      Photo by Richard Pasquarella on Unsplash
    </h4>
    <img
      alt="Photo by Richard Pasquarella on Unsplash"
      class="Card-image"
      src="image1.jpg"
    />
    <small
      class="Card-small"
    >
      Image 
       of 
      .
    </small>
  </div>
</div>
`;

exports[`works when you click on the right arrow 1`] = `
<div>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      Photo by Richard Pasquarella on Unsplash
    </h4>
    <img
      alt="Photo by Richard Pasquarella on Unsplash"
      class="Card-image"
      src="image1.jpg"
    />
    <small
      class="Card-small"
    >
      Image 
       of 
      .
    </small>
  </div>
</div>
`;
